www/js/app.js

```diff
@@ -914,7 +985,12 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
             var articleContent = iframeArticleContent.contentDocument.documentElement;
             articleContent.innerHTML = htmlArticle;
             // Add any missing classes stripped from the <html> tag
-            if (htmlCSS) articleContent.getElementsByTagName('body')[0].classList.add(htmlCSS);
+            var docBody = articleContent.getElementsByTagName('body')[0];
+            if (htmlCSS) docBody.classList.add(htmlCSS);
+            // Deflect drag-and-drop of ZIM file on the iframe to Config
+            docBody.addEventListener('dragover', handleIframeDragover, false);
```

Comment:
Why is there a last `false` argument here, where there isn't on line 841?

Reply:
The code I was using as a model had these. I hadn't actually checked what it does. I see false means "The event handler is executed in the bubbling phase". However, it's also the default! So I'll experiment with removing them.